MIGRATION_GUIDE.md

```diff
@@ -3,7 +3,23 @@
 
 This guide outlines how developers can migrate from older versions of our SDK to newer ones.
 
-## Migrating from v1.X.0 to v2.0.0
+## Migrating from v2.3.0 to v2.3.1
+It was recently discovered that the Swift SDK was using legacy event for some common event names like "Added to Cart", as well as some event names that were associated with server actions like "Product Ordered". This would result in some events being logged with names like "$added_to_cart" which is undesirable. In order to bring the Swift SDK in line with other product we have introduced new types to ensure the correct event name gets logged by default. The exmaple shows how to use the event names:
```

Comment:
I would tweak this slightly. I also noticed a few typos that need to be fixed.

It was recently discovered that the Swift SDK was using legacy event names for some common events, like "Added to Cart," and some events that are associated with server actions, like "Ordered Product." As a result, if your account used these standard events, they were being logged with names like "$added_to_cart," in contrast to website generated events which are logged as "Added to Cart." In order to bring the Swift SDK in line with Klaviyo's other integrations, we have introduced new types to ensure that the correct event name is logged by default. This example shows how to use the event names: